src/solver/conv_asm_implicit_gemm_gtc_fwd_nhwc.cpp

```diff
@@ -700,7 +700,8 @@ bool PerformanceConfigAsmImplicitGemmGTCFwdXdlopsNHWC::IsValid(
     if(!(tensor_a_thread_lengths[1] == 1 && tensor_b_thread_lengths[1] == 1))
     {
         // if both 1, indicate padded c support
-        if(((c >> gemm_k_global_split) / group) % gemm_k_per_block != 0)
+        if((c >> gemm_k_global_split == 0) ||
+           (((c >> gemm_k_global_split) / group) % gemm_k_per_block != 0))
```

Comment:
Does this fix correctness issue or something else?

Reply:
Yes. When `C` is larger than power 2 of `gemm_k_global_split`, the kernel should not be picked up. Hence if we do not have this condition, will have correctness issue.

Reply:
@atamazov I understand that `IsApplicable` is called after loading a Db entry, so a change to the databases is not required, can you please confirm?

Reply:
Interested in whether tuning is required after this PR too. @atamazov

Reply:
@JehandadKhan @junliume 

### Perf-db.
- If applicability is narrowed, then some records become unusable (harmless junk). Re-tuning is not required unless you want to remove the junk.
- If applicability is extended, then the solver is unable to show it's full potential. This is harmless as well. Re-tuning is needed only if we need to unleash the performance for the newly enabled configs ASAP.

### Find-db
- If applicability is narrowed, then some info become unusable as well (harmless junk), because we check applicability prior using info from find-db. Re-tuning is not required unless you want to remove the junk.
  - This is true since #1987 is complete. But if system find-db is is old format (this is true for 5.5 release), then performance drop may be bigger than expected (an unwanted side effect)
- If applicability is extended, then the solver may be unable to show it's full potential if the library runs in HYBRID or DYNAMIC_HYBRID modes, or when used in Immediate Mode.